sourcegraph/learn
# Update dependencies

### What should this PR do?
Resolves [DEVED-152](https://sourcegraph.atlassian.net/browse/DEVED-152) by updating dependencies from latest bot sweep in one place.

### Why are we making this change?
- Keeping the project up to date.

### What are the acceptance criteria? 
- The app should behave as it currently does on production.

### How should this PR be tested?
- Check out the branch or deploy preview and ensure that nothing looks out of place.

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.
